packages/@secretlint/secretlint-rule-basicauth/src/index.ts

```diff
@@ -35,7 +35,7 @@ function reportIfFoundBasicAuth({
     // https://developer.mozilla.org/docs/Web/HTTP/Authentication
     // https://ihateregex.io/expr/url
     const URL_PATTERN =
-        /(?<protocol>(:?[-a-zA-Z0-9_]{1,256})):\/\/(?<user>[-a-zA-Z0-9_]{1,256}):(?<password>[-a-zA-Z0-9_]{1,256})@[-a-zA-Z0-9%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b/g;
+        /(?<protocol>(?:https?|ftp|ftps)):\/\/(?<user>[-a-zA-Z0-9_]{1,256}):(?<password>[-a-zA-Z0-9_]{1,256})@[-a-zA-Z0-9%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b/g;
```

Comment:
@secretlint/secretlint-rule-basicauth and @secretlint/secretlint-rule-database-connection-string has same logics, but these treats differenet protocol.